tests/Signature/S3SignatureV4Test.php

```diff
@@ -115,6 +115,18 @@ public function testAddsSecurityTokenIfPresent()
         $this->assertContains('X-Amz-Expires=518400', $url);
     }
 
+    public function testUsesStartDateIfSpecified()
+    {
+        $options = ['start_time' => $_SERVER['aws_time']];
```

Comment:
The intent of the test would be clearer if you just used a literal value here instead of reading an environment variable. You need to go up to the setup method to see that `$_SERVER['aws_time']` is set to `strtotime('December 5, 2013 00:00:00 UTC')`, which is a layer of indirection that this PR makes unnecessary.